<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogoConsultas</class>
 <widget class="QWidget" name="DialogoConsultas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Realizar consulta</string>
  </property>
  <widget class="QTabWidget" name="opciones_consulta">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>471</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_consulta">
    <attribute name="title">
     <string>Consulta</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>741</width>
       <height>391</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="layout_consulta">
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupbox_periodo">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Periodo</string>
        </property>
        <widget class="QWidget" name="formLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>221</width>
           <height>161</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="layout_periodo">
          <item row="1" column="0">
           <widget class="QLabel" name="lbl_desde">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Desde</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDateEdit" name="dateedit_desde">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>10</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lbl_hasta">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Hasta</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDateEdit" name="dateedit_hasta">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>10</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_etiqueta">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Etiqueta</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineedit_etiqueta_periodo">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>10</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="btn_resetear_periodo">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Resetear periodo</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupbox_secciones">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Secciones</string>
        </property>
        <widget class="QListWidget" name="lista_secciones_en_consulta">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>221</width>
           <height>131</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Todas</string>
          </property>
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>10</pointsize>
            <italic>true</italic>
           </font>
          </property>
         </item>
        </widget>
        <widget class="QPushButton" name="btn_sacar_secciones">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>160</y>
           <width>169</width>
           <height>23</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Sacar secciones</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QGroupBox" name="groupbox_conceptos">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>Conceptos</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>221</width>
           <height>331</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="layout_concepto_consulta">
          <item>
           <widget class="QListWidget" name="lista_conceptos_en_consulta">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>10</pointsize>
              <italic>true</italic>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="btn_sacar_conceptos">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>360</y>
           <width>169</width>
           <height>23</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Sacar conceptos</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QGroupBox" name="groupbox_medios">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Medios</string>
        </property>
        <widget class="QListWidget" name="lista_medios_en_consulta">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>221</width>
           <height>131</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
           <italic>true</italic>
          </font>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_sacar_medios">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>160</y>
           <width>169</width>
           <height>23</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Sacar medios</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QGroupBox" name="groupbox_reportes">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Reportes</string>
        </property>
        <widget class="QListWidget" name="lista_reportes_en_consulta">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>221</width>
           <height>131</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Fuerza de concepto en medio</string>
          </property>
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>10</pointsize>
            <italic>true</italic>
           </font>
          </property>
         </item>
        </widget>
        <widget class="QPushButton" name="btn_agregar_reportes">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>160</y>
           <width>169</width>
           <height>23</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Sacar reportes</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QDialogButtonBox" name="btnbox_consulta">
     <property name="geometry">
      <rect>
       <x>411</x>
       <y>420</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_realizar_consulta">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>420</y>
       <width>120</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Realizar consulta</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_conceptos">
    <attribute name="title">
     <string>Conceptos</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>761</width>
       <height>431</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="layout_conceptos">
      <item>
       <widget class="QListWidget" name="lista_conceptos">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layout_opciones_conceptos">
        <item>
         <widget class="QPushButton" name="btn_agregar_conceptos">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Agregar conceptos</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_periodo">
    <attribute name="title">
     <string>Período</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>761</width>
       <height>431</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="layout_periodos">
      <item>
       <widget class="QListWidget" name="lista_periodos">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_opciones_periodos">
        <item>
         <widget class="QPushButton" name="btn_setear_periodo">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Setear periodo</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_medios">
    <attribute name="title">
     <string>Medios</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>9</y>
       <width>761</width>
       <height>431</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="layout_medios">
      <item>
       <widget class="QListWidget" name="lista_medios">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>false</italic>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layout_opciones_medios">
        <item>
         <widget class="QPushButton" name="btn_agregar_medios">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Agregar medios</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_secciones">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <attribute name="title">
     <string>Secciones</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>9</y>
       <width>761</width>
       <height>431</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="layout_secciones">
      <item>
       <widget class="QListWidget" name="lista_secciones"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layout_opciones_secciones">
        <item>
         <widget class="QPushButton" name="btn_agregar_secciones">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Agregar secciones</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_reporte">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <attribute name="title">
     <string>Reportes</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>761</width>
       <height>431</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="layout_reportes">
      <item>
       <widget class="QListWidget" name="lista_reportes"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layout_opciones_reportes">
        <item>
         <widget class="QPushButton" name="btn_sacar_reportes">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Agregar reportes</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <action name="action_agregar_conceptos">
   <property name="text">
    <string>agregar_conceptos</string>
   </property>
   <property name="toolTip">
    <string>agregar_conceptos</string>
   </property>
  </action>
  <action name="action_sacar_conceptos">
   <property name="text">
    <string>sacar_conceptos</string>
   </property>
  </action>
  <action name="action_setear_periodo">
   <property name="text">
    <string>setear_periodo</string>
   </property>
  </action>
  <action name="action_agregar_medios">
   <property name="text">
    <string>agregar_medios</string>
   </property>
  </action>
  <action name="action_agregar_secciones">
   <property name="text">
    <string>agregar_secciones</string>
   </property>
  </action>
  <action name="action_sacar_medios">
   <property name="text">
    <string>sacar_medios</string>
   </property>
  </action>
  <action name="action_sacar_secciones">
   <property name="text">
    <string>sacar_secciones</string>
   </property>
  </action>
  <action name="action_agregar_reportes">
   <property name="text">
    <string>agregar_reportes</string>
   </property>
  </action>
  <action name="action_sacar_reportes">
   <property name="text">
    <string>sacar_reportes</string>
   </property>
  </action>
  <action name="action_realizar_consulta_y_cerrar">
   <property name="text">
    <string>realizar_consulta_y_cerrar</string>
   </property>
  </action>
  <action name="action_resetear_periodo">
   <property name="text">
    <string>resetear_periodo</string>
   </property>
  </action>
  <action name="action_liberar_grafico">
   <property name="text">
    <string>liberar_grafico</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btn_agregar_conceptos</sender>
   <signal>released()</signal>
   <receiver>action_agregar_conceptos</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_setear_periodo</sender>
   <signal>released()</signal>
   <receiver>action_setear_periodo</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_agregar_medios</sender>
   <signal>released()</signal>
   <receiver>action_agregar_medios</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_agregar_secciones</sender>
   <signal>released()</signal>
   <receiver>action_agregar_secciones</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_sacar_reportes</sender>
   <signal>released()</signal>
   <receiver>action_sacar_reportes</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_realizar_consulta</sender>
   <signal>released()</signal>
   <receiver>action_realizar_consulta_y_cerrar</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnbox_consulta</sender>
   <signal>rejected()</signal>
   <receiver>DialogoConsultas</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_resetear_periodo</sender>
   <signal>released()</signal>
   <receiver>action_resetear_periodo</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_sacar_conceptos</sender>
   <signal>released()</signal>
   <receiver>action_sacar_conceptos</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_sacar_medios</sender>
   <signal>released()</signal>
   <receiver>action_sacar_medios</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>638</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_sacar_secciones</sender>
   <signal>released()</signal>
   <receiver>action_sacar_secciones</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_agregar_reportes</sender>
   <signal>released()</signal>
   <receiver>action_agregar_reportes</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>638</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
